docs/source/types.rst

```diff
@@ -0,0 +1,64 @@
+snowflake.snowpark.types
+========================
+This package contains all Snowpark logical types.
+
+.. currentmodule:: snowflake.snowpark.types
+
+   
+
+.. rubric:: Module Attributes
+
+.. autosummary::
+    :toctree: api/
+
+    Variant
```

Comment:
Could you move these two to the following section as well?

Reply:
I think these two refer to the type hint declaration: 
```
#: The type hint for annotating Variant data when registering UDFs.
Variant = TypeVar("Variant")

#: The type hint for annotating Geography data when registering UDFs.
Geography = TypeVar("Geography")
```
 In the [current API reference](https://docs.snowflake.com/en/developer-guide/snowpark/reference/python/_autosummary/snowflake.snowpark.types.html), these two are identified as module attributes as well.